### ISSUE TITLE
unbind() doesn't work

### ISSUE THREAD

**Issue by kaiserlos** [webcreate/infinite-ajax-scroll#157]
I want to unbind and restore pagination after a number of pages, otherwise it could become hard to get to my footer.

I have no idea why and also tried to directly unbind the scroll event from the window but nothing worked. Maybe you have an idea?

	 var ias = jQuery.ias({
	  container:  '.articles',
	  item:       '.item',
	  pagination: '.pagination',
	  next:       '.pagination li:last a'
	});

	

	// Adds a loader image which is displayed during loading
	ias.extension(new IASSpinnerExtension());

	// Adds a text when there are no more pages left to load
	ias.extension(new IASNoneLeftExtension({
	    text: 'Keine weiteren Anzeigen', // optionally
	    html: '<div class="ias-noneleft large-12 columns" style="text-align: center;">{text}</div>'
	}));

	ias.extension(new IASPagingExtension());
 
	ias.on('pageChange', function(pageNum, scrollOffset, url) {
		

	   /*
	    console.log(
	        "Welcome at page " + pageNum + ", " +
	        "the original url of this page would be: " + url
	    );

	    */
	   
	   if(pageNum >= 2) {
	   	ias.unbind();
	   }
	   
	});

**Comment by fieg** [comment]
@bukazz I released [v2.2.0](http://infiniteajaxscroll.com/blog/infinite-ajax-scroll-2-2-0-released.html) which should have fixed the unbind issues. With the new version you should be able to change/simplify your code to:

```js
function ajaxFilterEvents () {
  $.get(base_url + "ajax/filter-products", {uri_string: current_url, get_params: $.query.GET()}, function(data) {
    var obj = jQuery.parseJSON(data);  
    $('.inner-b').html(obj.items_list_html);

    $.ias().reinitialize(); // <--- new method in v2.2.0
  });
  var currentUrl = decodeURI(base_url + current_url + $.query.toString());
  history.pushState('data', '', currentUrl);
});
```

Would be great if you could verify this.

**Comment by fieg** [comment]
Closing due to no response.
